src/Build.UnitTests/Evaluation/ItemEvaluation_Tests.cs

```diff
@@ -650,5 +652,100 @@ public void MetadataPropertyFunctionBug()
 
             ObjectModelHelpers.AssertItems(new[] { $"{prefix}foo" }, items, expectedMetadata);
         }
+
+        [Fact]
+        public void ItemMetadataShouldBeEvaluatedCaseInsensitively()
+        {
+            var projectContents =
+            @"<Project>
+                <ItemGroup>
+                    <Foo Include=`Foo`>
+                        <SensitiVE>X</SensitiVE>
+                    </Foo>
+                </ItemGroup>
+                <Target Name=`AccessItemMetadata`>
+                    <CreateProperty Value=`%(Foo.FileName)=%(Foo.sensitive)`>  
+                        <Output TaskParameter=`Value` PropertyName=`Qualified` />  
+                    </CreateProperty>          
+                    <CreateProperty Value=`%(Foo.FileName)=%(sensitive)`>  
+                        <Output TaskParameter=`Value` PropertyName=`Unqualified` />  
+                    </CreateProperty>  
```

Comment:
Can you change this to use more-modern syntax like

```xml
<PropertyGroup>
 <Qualified>%(Foo.FileName)=%(Foo.sensitive)</Qualified>
 <Unqualified>%(Foo.FileName)=%(sensitive)</Unqualified>
</PropertyGroup>
```

Reply:
This one is funny. Structured log viewer got me so focused on always using tasks over shorthand property/itemgroups, because CreateProperty etc. give better binary logs. Maybe i should raise an issue over at https://github.com/KirillOsenkov/MSBuildStructuredLog :smile: